Modules/mmapmodule.c

```diff
@@ -29,6 +29,8 @@
 #include "structmember.h"         // PyMemberDef
 #include <stddef.h>               // offsetof()
 
+#if !defined(MS_WINDOWS) || defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_SYSTEM) || defined(MS_WINDOWS_GAMES)
```

Comment:
We can't support the system API partition until `mmap` is updated to call `CreateFileMappingW()` and `OpenFileMappingW()`. As of 10.0.19041, "winbase.h" declares the ANSI versions only for desktop and games.

```suggestion
#if !defined(MS_WINDOWS) || defined(MS_WINDOWS_DESKTOP) || defined(MS_WINDOWS_GAMES)
```